src/python/grpcio/grpc/_runtime_protos.py

```diff
@@ -0,0 +1,126 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+
+def _uninstalled_protos(*args, **kwargs):
+    raise NotImplementedError(
+        "Install the grpcio-tools package to use the protos function.")
+
+
+def _uninstalled_services(*args, **kwargs):
+    raise NotImplementedError(
+        "Install the grpcio-tools package to use the services function.")
+
+
+def _uninstalled_protos_and_services(*args, **kwargs):
+    raise NotImplementedError(
+        "Install the grpcio-tools package to use the protos_and_services function."
+    )
+
+
+def _interpreter_version_protos(*args, **kwargs):
+    raise NotImplementedError(
+        "The protos function is only on available on Python 3.X interpreters.")
+
+
+def _interpreter_version_services(*args, **kwargs):
+    raise NotImplementedError(
+        "The services function is only on available on Python 3.X interpreters."
+    )
+
+
+def _interpreter_version_protos_and_services(*args, **kwargs):
+    raise NotImplementedError(
+        "The protos_and_services function is only on available on Python 3.X interpreters."
+    )
+
+
+def protos(protobuf_path):  # pylint: disable=unused-argument
+    """Returns a module generated by the indicated .proto file.
+
+    THIS IS AN EXPERIMENTAL API.
+
+    Use this function to retrieve classes corresponding to message
+    definitions in the .proto file.
+
+    To inspect the contents of the returned module, use the dir function.
+    For example:
+
+    ```
+    protos = grpc.protos("foo.proto")
+    print(dir(protos))
+    ```
+
+    The returned module object corresponds to the _pb2.py file generated
+    by protoc. The path is expected to be relative to an entry on sys.path
+    and all transitive dependencies of the file should also be resolveable
+    from an entry on sys.path.
+    """
+
+
+def services(protobuf_path):  # pylint: disable=unused-argument
+    """Returns a module generated by the indicated .proto file.
+
+    THIS IS AN EXPERIMENTAL API.
+
+    Use this function to retrieve classes and functions corresponding to
+    service definitions in the .proto file, including both stub and servicer
+    definitions.
+
+    To inspect the contents of the returned module, use the dir function.
+    For example:
+
+    ```
+    services = grpc.services("foo.proto")
+    print(dir(services))
+    ```
+
+    The returned module object corresponds to the _pb2_grpc.py file generated
+    by protoc. The path is expected to be relative to an entry on sys.path
+    and all transitive dependencies of the file should also be resolveable
+    from an entry on sys.path.
+    """
+
+
+def protos_and_services(protobuf_path):  # pylint: disable=unused-argument
+    """Returns a 2-tuple of modules corresponding to protos and services.
+
+    THIS IS AN EXPERIMENTAL API.
+
+    The return value of this function is equivalent to a call to protos and a
+    call to services.
```

Comment:
nit: Should we use the format of `Args:` and `Returns:` to describe these 3 new methods?

Reply:
Also, we could document `GRPC_PYTHON_DISABLE_DYNAMIC_STUBS` here. Otherwise, people have to read source code to find this environment variable.

Reply:
Done.